packages/docgen/lib/get-leading-comments.js

```diff
@@ -14,7 +9,10 @@ const { last } = require( 'lodash' );
 module.exports = ( declaration ) => {
 	let comments;
 	if ( declaration.leadingComments ) {
-		const lastComment = last( declaration.leadingComments );
+		const lastComment =
+			declaration.leadingComments[
```

Comment:
If `declaration.leadingComments` is a plain array, `declaration.leadingComments.at(-1)` would also work, assuming that the transpilation infrastructure supports it. But it may be safer to leave it as-is, and it's perfectly readable this way as well.

Reply:
I previously tried it, and it broke 😉 Seems like it's not supported.